src/packages/pantheon-apps.yaml

```diff
@@ -1,15 +1,13 @@
 packages:
- - com.github.stsdc.monitor
  #- elementary-calculator
  #- elementary-calendar
  #- elementary-camera
  #- elementary-capnet-assist
  - elementary-code
- - elementary-mail
+ #- elementary-mail
  - elementary-music
  - elementary-photos
  #- elementary-screenshot-tool
  - elementary-sideload
- - elementary-tasks
+ #- elementary-tasks
```

Comment:
This package doesn't exist in Terra and needs to be commented out

Reply:
- shouldn't that be covered by the changes you already did? looks to be fine https://github.com/pieckenst/sodalite/blob/release/next/src/packages/pantheon-apps.yaml ?